src/NationalInstruments.Analyzers/Properties/Resources.resx

```diff
@@ -345,4 +345,16 @@
   <data name="IdentifiersShouldBeSpelledCorrectlyTitle" xml:space="preserve">
     <value>Identifiers should be spelled correctly</value>
   </data>
+  <data name="NI1017_Description" xml:space="preserve">
+    <value>If you invoke a chain of methods and/or properties and that single line becomes very long, split each invocation onto its own line.</value>
+    <comment>An optional longer localizable description of the diagnostic.</comment>
+  </data>
+  <data name="NI1017_Message" xml:space="preserve">
+    <value>Split long chains of dotted method/property invocations. Use well-named local variables if needed.</value>
```

Comment:
Are we going to enforce this or make it "info" severity? Maybe prefix with "Consider"?

Reply:
Actually, we probably want to promote it to an enforcement. I think that was one of the reasons I started writing this analyzer, the last I talked with Jayson.

@jryckman can you confirm?

Reply:
It's part of our  conventions, so it's okay to default to warning

Reply:
Fixed in the latest update.